styles/themes/theme.tsx

```diff
@@ -239,6 +240,14 @@ theme.components = {
       },
     },
   },
+  MuiSelect: {
+    styleOverrides: {
+      icon: {
+        fontSize: '3rem',
+        color: '#2f3554',
```

Comment:
```suggestion
        color: theme.palette.primary.dark,
```
I noticed that we already have that color `#2F3554` on our pallete, so what do you think about change that hardcoded one to one using our palette?

Reply:
Ahh I thought I changed that I noticed that as well will make change.